docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -0,0 +1,971 @@
+
+:::
+
+You are preparing everything for the newly created region to take over again to restore the benefits of a dual-region setup.
+
+For this, you need to stop the Zeebe exporters to not export any new data to Elasticsearch, so you can create a backup.
+
+Additionally, you need to scale down Operate and Tasklist. This will result in users not being able to interact with the Camunda Platform anymore and is required to guarantee no new data is imported to Elasticsearch.
+
+:::note
+
+That this **does not** affect processing of process instances in any way. The impact is that some information about the affected instances might not be visible in Operate.
+
+:::
+
+#### How to get there
+
+1. Disable Operate and Tasklist by scaling to 0
+
+```bash
+OPERATE_DEPLOYMENT=$(kubectl --context $CLUSTER_0 get deployment --selector=app\.kubernetes\.io/component=operate -o jsonpath='{.items[0].metadata.name}' -n $CAMUNDA_NAMESPACE_0)
+TASKLIST_DEPLOYMENT=$(kubectl --context $CLUSTER_0 get deployment --selector=app\.kubernetes\.io/component=tasklist -o jsonpath='{.items[0].metadata.name}' -n $CAMUNDA_NAMESPACE_0)
+
+kubectl --context $CLUSTER_0 scale deployments/$OPERATE_DEPLOYMENT --replicas 0
+kubectl --context $CLUSTER_0 scale deployments/$TASKLIST_DEPLOYMENT --replicas 0
+
+```
+
+2. Disable the Zeebe Elasticsearch exporters in Zeebe via kubectl
+
+```bash
+ZEEBE_GATEWAY_SERVICE=$(kubectl --context $CLUSTER_0 get service --selector=app\.kubernetes\.io/component=zeebe-gateway -o jsonpath='{.items[0].metadata.name}' -n $CAMUNDA_NAMESPACE_0)
+kubectl --context $CLUSTER_0 port-forward services/$ZEEBE_GATEWAY_SERVICE 9600:9600 -n $CAMUNDA_NAMESPACE_0
+curl -i localhost:9600/actuator/exporting/pause -XPOST
+# The successful response should be:
+# HTTP/1.1 204 No Content
+```
+
+#### Verification
+
+For Operate and Tasklist, you can confirm that the deployments have successfully scaled down by listing those and indicating `0/0` ready.
+
+```bash
+kubectl --context $CLUSTER_0 get deployments $OPERATE_DEPLOYMENT $TASKLIST_DEPLOYMENT -n $CAMUNDA_NAMESPACE_0
+# NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+# camunda-operate    0/0     0            0           23m
+# camunda-tasklist   0/0     0            0           23m
+```
+
+For the Zeebe Elasticsearch exporters, there's currently no API available to confirm this. Only the response code of `204` indicates a successful disabling.
+
+</div>
+  </TabItem>
+  <TabItem value="step3" label="Step 3">
+
+#### Create and Restore Elasticsearch Backup
+
+<StateContainer
+current={<Ten viewBox="140 40 680 500" />}
+desired={<Eleven viewBox="140 40 680 500" />}
+/>
+
+<div>
+
+#### Current
+
+The Camunda Platform is currently not reachable by end-users and does not process any new processes to allow creating a backup of Elasticsearch without losing any new data.
```

Comment:
One suggestion in the past was that we also scale down the gateway.

Reply:
We have not currently included it in the tests but we could consider disabling the gateway during the backup and restore phase. I'd assume it doesn't do much harm since it just limits the outside to reach Zeebe.
Let me know what you think